torch_geometric/transforms/random_link_split.py

```diff
@@ -40,7 +40,8 @@ class RandomLinkSplit(BaseTransform):
             (default: :obj:`0.2`)
         is_undirected (bool): If set to :obj:`True`, the graph is assumed to be
             undirected, and positive and negative samples will not leak
-            (reverse) edge connectivity across different splits.
+            (reverse) edge connectivity across different splits. Note that
```

Comment:
Maybe something like "Note that this only affects the graph split, label data will not be returned undirected". I think its good to be clear about what isn't done.